packages/block-editor/src/components/rich-text/index.native.js

```diff
@@ -220,6 +218,7 @@ function RichTextWrapper(
 				selectionChangeEnd
 			);
 		},
+		// eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Because the Eslint rule was added after this code was introduced. Since I had to edit some lines in this file, I couldn't commit it (without skipping the commit hooks). I don't feel confident introducing any changes related to `onSelectionChange` / `onEnter` as adding newer dependencies could cause side effects we don't expect.

I think that change should happen with any possible refactors we might do for RichText.

Reply:
> but the worst case of adding the dependency would be an extra re-render, WDYT?

Correct, but that could take a toll on the performance of the editor if we are adding any unneeded re-renders so I'd say let's keep `exhaustive-deps` temporarily.